docs/DevDocs/SatelliteImages.md

```diff
@@ -0,0 +1,92 @@
+
+## Collections
+
+RD-WATCH will query the STAC server for 4 sources:
+
+- WV: WorldView
+- S2: Sentinel 2
+- L8: Landsat 8
+- PL: Planet Labs
+
+WorldView is treated differently from the rest of Satellite Sources.  WorldView can be pansharpened if there exists additional imagery data.  This allows a higher resolution image to be displayed when compared to the other Sources.
+
+There is an environment variable `RDWATCH_ACCENTURE_VERSION` that is used to determine the collections that are searched when querying the STAC server.
+
+## Code Layout
+
+The [**pystac**](https://pystac.readthedocs.io/en/stable/) library is utilized to make queries to the STAC server.
+
+Within the **./rdwatch/core/utils** directory contains helper tools for Querying the STAC server and processing the resulting images.
+
+In the root of that folder is a `stac_search.py` which is used for S2,L8,PL image sources.
+The Collections to search for images are specified [here](https://github.com/ResonantGeoData/RD-WATCH/blob/58ab08d3d5ef7905295b9cec53b0e7a73c0dd5d6/rdwatch/core/utils/stac_search.py#L59-L62)
+
+```
+if settings.ACCENTURE_VERSION is not None:
+    COLLECTIONS['S2'].append(f'ta1-s2-acc-{settings.ACCENTURE_VERSION}')
+    COLLECTIONS['L8'].append(f'ta1-ls-acc-{settings.ACCENTURE_VERSION}')
+    COLLECTIONS['PL'].append(f'ta1-pd-acc-{settings.ACCENTURE_VERSION}')
+```
+
+### Querying STAC
+
+Within **./rdwatch/core/utils/satellite_bands.py** file is the main function `get_bands`.  This function performs a STAC Query for a time range and a bounding box utilizing **pystac**.  The results are typically verbose so it is paginated with 100 results at a time.  It iterates over these times and returns all 'visual' images that match the query.  If the results match a visual band image they are returned as a `Band` class.  This `Band` class contains the following information:
+
+- **constellation**: source of image (S2, L8, PL)
+- **spectrum**: Information about the spectrum the band captures (μm)
+- **level**: processing level
+- **timestamp**: date of the satellite image
+- **bbox**: bounding box associated with the image
+- **uri**: Typically contains the S3 URL for the COG that contains the image
+- **cloudcover**: optional value that will extract metadata to indicate the amount of cloud cover in the image
+- **collection**: The collection this image was retrieved from
+
+### WorldView Differences
+
+Within the **./rdwatch/core/utils** folder there are two folders labeled **worldview**.
+
+The base **worldview** folder is an older folder that was used for just rendering the source WorldView Images.
+The **worldview_processed** folder contains tools to create a pansharpened version of the image buy downloading extra data.
```

Comment:
The `worldview` directory was removed in this PR.

```suggestion
Within the **./rdwatch/core/utils** folder there is a **worldview_processed** folder. It contains tools to create a pansharpened version of worldview images by downloading extra data.
```

Reply:
Good call, forgot to update the docs after removing it.